src/routes/Search/containers/SearchContainer.js

```diff
@@ -0,0 +1,24 @@
+import {connect} from 'react-redux'
+import {createSearch} from '../modules/search'
+
+/*  This is a container component. Notice it does not contain any JSX,
+ nor does it import React. This component is **only** responsible for
+ wiring in the actions and state necessary to render a presentational
+ component - in this case, the counter:   */
+
```

Comment:
Search container needs to be a _class_ with
- wiring to redux and redux-form
- validation (if not in an external util)
- render method that passes props (e.g. `<Search {...this.props} />`) and renders Search View